libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -1182,10 +1178,6 @@ void EthStratumClient::processResponse(Json::Value& responseObject)
             {
                 std::string enonce = jPrm.get((Json::Value::ArrayIndex)0, "").asString();
                 processExtranonce(enonce);
-                if (m_onWorkReceived && (m_current.epoch != -1))
```

Comment:
Eth stratum spec says that extra nonce updates apply to the next job, not the current job. Do not resubmit the current job.